src/Tribe/Views/V2/View.php

```diff
@@ -1590,6 +1590,13 @@ public function get_today_url( $canonical = false ) {
 		// Handle the `eventDisplay` query arg due to its particular usage to indicate the mode too.
 		$query_args['eventDisplay'] = $this->slug;
 
+		$category = $this->context->get( 'event_category', false );
+
+		if ( is_array( $category ) ) {
+			$category = Arr::to_list( reset( $category ) );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 23 spaces but found 1 space

<sub>hash: bee87ac7e9f4cae7c9768426db1ed073</sub>